nexus/db-model/src/vpc_firewall_rule.rs

```diff
@@ -211,20 +211,55 @@ pub struct VpcFirewallRule {
     pub priority: VpcFirewallRulePriority,
 }
 
+/// Cap on the number of rules in a VPC
+///
+/// The choice of value is somewhat arbitrary, but the goal is to have a
+/// large number that customers are unlikely to actually hit, but which still
+/// meaningfully limits the ability to overload the DB with a single request.
+const MAX_FW_RULES_PER_VPC: usize = 1024;
+
+/// Cap on targets and on each type of filter
+const MAX_FW_RULE_PARTS: usize = 128;
```

Comment:
I took a look at GCP's limits https://cloud.google.com/firewall/docs/quotas#per-firewall-rule . Perhaps we should increase this one, or make it more nuanced?

Reply:
I’d rather just make it bigger. I’d want to see what people’s needs are before introducing a more granular set of limits that aren’t grounded in any real limitations of the system.

Reply:
I bumped the parts to 256.